src/main/java/com/redhat/devtools/intellij/tektoncd/utils/CRDHelper.java

```diff
@@ -20,9 +20,11 @@
     public static CustomResourceDefinitionContext getCRDContext(String apiVersion, String plural) {
         if (Strings.isNullOrEmpty(apiVersion) || Strings.isNullOrEmpty(plural)) return null;
         String[] groupVersion = apiVersion.split("/");
-        if (groupVersion.length != 2) return null;
+        if (groupVersion.length != 2 && !apiVersion.equalsIgnoreCase("triggers.tekton.dev")) {
+            return null;
+        }
         String group = groupVersion[0];
-        String version = groupVersion[1];
+        String version = groupVersion.length > 1 ? groupVersion[1]: "v1alpha1";
```

Comment:
You claimed we switched to v1beta1 so shouldn't it be v1beta1 ?

Reply:
Yes but not all resources switched to beta, only task, taskrun, pipeline, pipelinerun. Resources are still and always be alpha because they will be removed soon. Conditions are still alpha bc they are new. Triggers still alpha, i guess for the same reason as conditions. 

I added this control because any resource in beta will explicitly declare that in its version (otherwise we are not sure which one to use alpha or beta), but if it's not explicitly declared it will be alpha for sure. An example are triggers. Idk the behavior in minishift but in CRC triggers will have version "triggers.tekton.dev". Even if you add a resource using "triggers.tekton.dev/v1alpha1" it will be renamed to "triggers.tekton.dev"